index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Page Title</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" href="css/style.css" />
+	<base target="_blank">
+</head>
+<body>
+	<div class="container">
+		<div class="header">
+			<div class="gretting">
+				<h1>Hello!<br>
+					My Name is<br>
+					Drobovich Vladislav
+				</h1>
+			</div>
+			<div class=photo>
+				<img src="resources/photo.jpg" alt="image of a man" width="300" height="300">
+			</div>
+			<div class="contacts">
+				<img class="icon-contacts" src="resources/vk.png" alt="icon vk">
+				<h3 class="contacts-text"><a href="https://vk.com/vlad_drobovich" target="_blank">vk.com/vlad_drobovich</a></h3>
+				<img class="icon-contacts" src="resources/telegram.png" alt="icon telegram">
+				<h3 class="contacts-text">+375333625874</h3>
+				<img class="icon-contacts" src="resources/mail.png" alt="icon mail">					<h3 class="contacts-text">vladdrobovich@gmail.com</h3>
+			</div>
+		</div>
+		<div class="about">
+			<h2>About me</h2>
+			<img class="icon-about" src="resources/js.png" alt="icon js">
```

Comment:
If you have repeatable items it is better to wrap them in common element, as example: `<div class="about-block">` which will contain image and paragraph. It will be simpler to stylize elements relatively to this container.